src/Files.App/Views/ShellPanesPage.xaml.cs

```diff
@@ -356,6 +357,23 @@ public void FocusOtherPane()
 				GetPane(0)?.Focus(FocusState.Programmatic);
 		}
 
+		/// <inheritdoc/>
+		public void UpdateOpenPanesPreferences(string targetPath, bool includeActivePane = false)
+		{
+			foreach (var pane in GetPanes())
```

Comment:
I'm not certain this is doable, but it might be worth seeing if this code could be put in the layout helper as well.

Reply:
I'll see what I can do

Reply:
I just had a look at `ModernShellPage` implementation and it seems there are no "dangerous" public things, so I guess we can make it public